apps/condo/domains/common/components/EmptyListView.tsx

```diff
@@ -62,13 +63,24 @@ export const EmptyListView: React.FC<IEmptyListProps> = ({ label, message, creat
             <Typography.Text type={'secondary'}>
                 {message}
             </Typography.Text>
-            <Button
-                type={'sberDefaultGradient'}
-                style={{ marginTop: '24px' }}
-                onClick={() => router.push(createRoute)}
-            >
-                {createLabel}
-            </Button>
+            <Row gutter={[10, 0]} align={'middle'} justify={'center'} style={{ marginTop: '24px' }}>
```

Comment:
You should move all styles to constants (align and justify too)

It is also recommended to take out the gutter

const _ : RowProps['gutter']